java-api/src/main/java/com/sap/cloud/security/token/AccessToken.java

```diff
@@ -50,6 +52,21 @@
 	@Nullable
 	GrantType getGrantType();
 
+	/**
+	 * Returns subaccount identifier. This reflects claim {@code ext_attr.subaccountid} in xsuaa access tokens.
+	 * For example, commercialized multi-tenant applications with a need for metering and billing use
+	 * {@link #getSubaccountId()} method as identifier for the account to be billed.<br>
+	 *
+	 * Multi-tenant applications need to adapt by using the zone ID instead of the subaccount ID as key
```

Comment:
I would suggest to remove "by" like this:
Multi-tenant applications need to adapt using the zone ID instead of the subaccount ID as key